content/SCALE/GettingStarted/SCALEReleaseNotes.md

```diff
@@ -55,26 +52,53 @@ More details are available from [Software Releases]({{< relref "/TrueNASUpgrades
 
 * {{< include file="/static/includes/UpgradeClearCache.md" >}}
 
-* As of [June 6, Kubernetes is removed from 24.10 Nightly builds](https://forums.truenas.com/t/apps-update-2024-06-06/6041).
-  The apps feature is not functional in these nightly builds while development continues with replacing Kubernetes with Docker Compose ([announcement](https://forums.truenas.com/t/the-future-of-electric-eel-and-apps/5409)).
+* 24.10 moves the applications backend from Kubernetes to Docker ([announcement](https://forums.truenas.com/t/the-future-of-electric-eel-and-apps/5409)).
+
+  * Supported catalog applications automatically migrate to Docker deployments on upgrade from from 24.04 (Dragonfish) to Electric Eel.
+  
+    App migration is receiving active development in 24.10-BETA.1.
+    To see which applications currently support automatic migration, see the [Parity Status with truenas/charts](https://github.com/truenas/apps?tab=readme-ov-file#parity-status-with-truenascharts) chart from the /truenas/apps/ github repository.
+    Applications that support migration display a green check (✅) in both **Added** and **Migrated** columns.
+    Check back regularly and note the update history of the [README.md](https://github.com/truenas/apps/blob/master/README.md) file for the latest developments.
+
+    Configuration data for applications that do not automatically migrate is retained in the ixapplications dataset.
```

Comment:
This is now the ix-apps dataset
```suggestion
    Configuration data for applications that do not automatically migrate is retained in the ix-apps dataset.
```

Reply:
keep this one because that is the previously installed applications addressed in the next sentence